frontend/src/routes/UserManagement/RoleAssignments/RoleAssignmentModal.tsx

```diff
@@ -25,84 +24,48 @@ type RoleAssignmentModalProps = {
 const RoleAssignmentModal = ({ close, isOpen, isEditing, preselected }: RoleAssignmentModalProps) => {
```

Comment:
all about modal will be removed, but I decided to refactor at least for testing purposes